bddl/activity_definitions/make_a_small_vegetable_garden/problem0.bddl

```diff
@@ -23,9 +23,9 @@
         (ontop soil__bag.n.01_1 floor.n.01_1)
         (filled pumpkin_seed__bag.n.01_1 pumpkin_seed.n.01_1)
         (ontop pumpkin_seed__bag.n.01_1 coffee_table.n.01_1) 
-        (ontop pottable__chili.n.01_1 coffee_table.n.01_1) 
-        (ontop pottable__beefsteak_tomato.n.01_1 coffee_table.n.01_1)
-        (ontop pottable__beefsteak_tomato.n.01_2 coffee_table.n.01_1)
+        (ontop chili.n.01_1 coffee_table.n.01_1) 
+        (ontop beefsteak_tomato.n.01_1 coffee_table.n.01_1)
+        (ontop beefsteak_tomato.n.01_2 coffee_table.n.01_1)
```

Comment:
we should probably either 1) keep the pottable (so a small tree), or 2) use beefsteak_tomato_seed.n.01 or something like this